pkg/controller/controller.go

```diff
@@ -145,31 +145,31 @@ var (
 	}
 
 	provisionerSecretParams = secretParamsMap{
-		name:                         "Provisioner",
+		name: "Provisioner",
```

Comment:
Did you mean to change this indentation? I'm pretty sure `gofmt` will add this back when its ran

Reply:
I run gofmt in normal format, but the file is indented when I save it